docs/platforms/react-native/user-feedback/configuration/index.mdx

```diff
@@ -7,14 +7,18 @@ description: "Learn about the User Feedback Widget configuration options."
 The User Feedback Widget offers many customization options, and if the available options are insufficient, you can [use your own UI](/platforms/react-native/user-feedback/#user-feedback-api).
 
 ![An image showing the main customization options for the User Feedback Widget](./img/mobile-user-feedback-widget-customization.png)
-To collect user feedback from inside your application use the `showFeedbackWidget` method.
+To collect user feedback from inside your application use the `showFeedbackButton`/`hideFeedbackButton` to show/hide a button that triggers the Feedback Widget or the `showFeedbackWidget` method to present the widget directly.
```

Comment:
NIT: just a small comma that could help with reading 
```suggestion
To collect user feedback from inside your application, use the `showFeedbackButton`/`hideFeedbackButton` to show/hide a button that triggers the Feedback Widget or the `showFeedbackWidget` method to present the widget directly.
```

Reply:
Memo:
Mentioned functions are exported here: https://github.com/getsentry/sentry-react-native/blob/4492611acf91d678845981d572ed69c40178fcb3/packages/core/src/js/index.ts#L92-L93